### ISSUE TITLE
Narratives / Analysis

### ISSUE THREAD

**Issue by noleti** [neherlab/covid19_scenarios#231]
# 🙋 Feature Request

I really like the "narratives" of nextstrain.org (which I understand shares project members with this project). In particular, small writeups of experts analyzing particular datasets or aspects. I realized similar things might be possible with the covid19_scenarios, now that exporting/ URLs with configurations are supported. Maybe some people already did similar writeups elsewhere? 

I know everyone is super busy, but in the medium term it would be nice if either a) narratives are directly supported by the app or featured on the main website, and/or b) links to other peoples analysis based on the simulator could be provided. 

## 🔦 Context

Narratives could a) provide an easy entry for people to understand the app, b) provide additional insights by experts, c) increase the reach of the app via social media etc. 

## 😯 Describe the feature

If possible, provide some support for narratives through a) hosting and mechanisms to submit narratives, b) allow third parties to integrate the main plot or other elements in their page?

## 💻 Examples

An example narrative could look into how well the spread of covid-19 in a country matches the model, and discuss where mismatch could come from (e.g, insufficient testing, infection of particular age groups, simplifications in the model), possibly even short-term predictions etc. Can we see impact of mitigations in the real data, how does it match the simulation, etc?

I realize that some of this might be speculation, and hosting or linking third-party analysis might impose risks. 

Alternatively, third parties could configure some scenario on the app, and then obtain a way/URL/API that allows them to generate a specific plots and embed them into their page, e.g. the main visualization (as it has most interactive features).

## 💁 Possible Solution

To self-host narratives:
- you did great with nextstrain.org, so it is unlikely I can give you great ideas here
- As all of you are likely busy, maybe others are interested to contribute (not me, I'm not an expert)

For links to third parties:
- Just some (commented) link collection? If people write something that uses the app, they could let you know? 

For third-party integration:
- Provide APIs or paths in the app that will render only specific parts of the simulation(e.g. results), instead of the whole page. Ideally those could be embedded in third party sites

## Related

 -
 -

**Comment by ivan-aksamentov** [comment]
@noleti Yep, I've been thinking about it too.
If we venture into server territory, we might have other people's results available fpor viewing. Possibly maybe with some chat/comment-style discussion attached.

However, I'don't know if we'd have time for full-blown narratives.
If we will can we count on your participation in terms of user experience ideas and the actuall writeups?

**Comment by lizzwestman** [comment]
**Suggestion:** 

Has anyone written out/mapped the storytelling version of the arc and flu narrative for those who have survived the infection from start to finish? It should be there for a baseline from those who are confirmed to have it. 

It also should include versions from those assumed to have it but have not had access to tests yet --> **THIS PART HAS TO BE CLEARLY STATED AS A PRESUMPTION ONLY** that it is ASSUMED BY THOSE REPORTING THEIR ARC to be not just a flu, but due to access and options can neither be confirmed or denied. These should also not be easily accessed to prevent panic, just used as point of reference when comparing timelines as access to testing becomes more widespread. 

**### All points need to be vetted by actual medical professionals and use literal and real medical-quality scientific methodologies before being accepted as narratives.** 

Storytelling can be as important as data, especially in unknown situations and incomplete data like these situations can find better spots to test hypotheses if alignments are found through narratives. 

**Needed for narratives/storytelling (more details TK):**
- timeline 
- symptoms
-- details of symptoms and timeline/progression 
-- details of symptoms and timeline/healing (regression?) 
- general flu progression
- emotional mapping if possible and with an established taxonomy and has significant room for specifics and variances towards and away from the agreed taxonomy 
- general patient history overview of pre-existing as personal without violating any HIPAA or personal information data rules applicable to financial or whatever 
- geographic location 
- age 
- travel history since October 
- TBD
- TBD


Half-done narratives might be as dangerous as reporting half-done data. Please PLEASE be careful with this project. 

**Misinformation kills and panic kills at high rates too.**

**Comment by noleti** [comment]
Right, in my mind it would be less about symptoms of covid-19, and less about personal stories (but that could be up to the person writing this/publishing this on their site). The severity assumptions should be mentioned - i.e. only 5% or all 10-19yo are assumed to be confirmed, for example (somewhat relating to testing availability). The delay between infection and symptoms/infectious period could be mentioned, and R0. Impact of mitigation on R0. While doing this, values for a specific country/region could be discussed and the impact of changes to variables could be shown.

Like I stated above, there are technical and a writing aspects here. Maybe this project should focus more on providing technical means, i.e. ways to embed the interactive plot into a third party side (with parameters provided by URL).

**Comment by nnoll** [comment]
@lizzwestman I think @noleti was using narratives as a slightly more technical version of the term than is typically used colloquially. Narratives, as used by nexstrain.org, are a way in which researchers can create curated explorations of a given dataset, within the context of the tools of the site, to highlight particular features to a broad audience. It is not about hosting stories of those who were infected, which I can categorically say is not within the scope of this project (not say this is not important, however this is best done by more qualified individuals). Please let me know if I misinterpreted your post.

I'll just add that I think this is a great idea and will become more relevant as more data is added to our model.
